pkg/github/github.go

```diff
@@ -34,7 +34,7 @@ func GenerateNewRepositoryName(displayName, namespace, clusterName string) strin
 	sanitizedName := util.SanitizeName(displayName)
 	h := sha256.New()
 	h.Write([]byte(clusterName + namespace))
-	namespaceClusterHash := string(h.Sum(nil))[0:5]
+	namespaceClusterHash := string(h.Sum(nil))
```

Comment:
I thought we wanted to truncate the name?

Reply:
🤦 That was a mistake

Reply:
Fixed!